specs/posix/docker_container_mounts.table

```diff
@@ -3,7 +3,7 @@ description("Docker container mounts.")
 schema([
     Column("id", TEXT, "Container ID", index=True),
     Column("type", TEXT, "Type of mount (bind, volume)"),
-    Column("name", TEXT, "Optional mount name"),
+    Column("name", TEXT, "Optional mount name", index=True),
```

Comment:
Is this `name` unique (e.g., a pkey?) or could the same `name` show up in different container IDs?

Reply:
I was super unsure on these. There's a bunch of indirection in these tables